microprofile/WFLY-12737_EAP7-1368_quickstart-for-mp-health-2.1.adoc

```diff
@@ -0,0 +1,62 @@
+= WFLY-12737 / EAP7-1368 Quickstart for MP Health 2.1
+:author:            Martin Stefanko
+:email:             mstefank@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+This RFE provides a new quickstart and guide for MicroProfile Health 2.1 use in WildFly /
+JBoss EAP. The example will cover all basic functionalities of MicroProfile Health 2.1,
+namely: liveness and readiness checks, the basic usage of the `HealthCheckResponseBuilder`,
+and the inclusion of user specific data in the health check responses.
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.jboss.org/browse/EAP7-1368[EAP7-1368]
+* https://issues.jboss.org/browse/WFLY-12737[WFLY-12737]
+
+=== Dev Contacts
+
+* mailto:mstefank@redhat.com[Martin Stefanko]
+
+=== QE Contacts
+
+* TBD
+
+=== Testing By
+// Put an x in the relevant field to indicate if testing will be done by Engineering or QE. 
+// Discuss with QE during the Kickoff state to decide this
+[x] Engineering
```

Comment:
Any reason for this to not tested by QE?

Reply:
still waiting for a tester to be assigned. I will adjust this value after that.

Reply:
That doesn't sound like a good idea - if you specify that you are going to test it, QE will not be looking for assignee.

Reply:
after the discussion with QE. This RFE will be tested by engineering.